src/components/Home.vue

```diff
@@ -42,7 +42,7 @@ export default {
     }
   },
   async created () {
-    const response = await axios.get('http://api.fullstackers.io/v0/presentations')
+    const response = await axios.get('https://api.fullstackers.io/v0/presentations')
```

Comment:
It may be a good idea to pull out instances of domain names (at the very least ours) to a config.js file so that if we ever need to change part of it (e.g. changing `/v0/` to `/v1/` some day) we can do it in one spot.

Reply:
I agree and will research the best practice for implementing a config file in Vue.js.